packages/relay/tests/lib/eth.spec.ts

```diff
@@ -97,6 +97,11 @@ describe('Eth', async function () {
     expect(result).to.eq(false);
   });
 
+  it('should execute "eth_blockNumber"', async function () {
+    const blockNumber = await Relay.eth().blockNumber();
```

Comment:
You should actually mock the mirrorNode response for this case.
That's why it's failing.
Please see `mirrorNodeClient.spec.ts` for examples